godotcord_network_peer.cpp

```diff
@@ -763,3 +764,7 @@ NetworkedMultiplayerGodotcord::~NetworkedMultiplayerGodotcord() {
 		close_connection();
 	}
 }
+
+List<NetworkedMultiplayerGodotcord::GodotcordPeer> get_connected_peers() {

```

Comment:
get_connected_peers() declaration starts here. I don't know why git said everything else got changed for some reason.